cmake/add_pstore.cmake

```diff
@@ -266,6 +266,47 @@ function (add_pstore_executable target_name)
 endfunction(add_pstore_executable)
 
 
+############################
+# add_pstore_install_targets
+############################
+
+function(add_pstore_install_targets target)
+  cmake_parse_arguments(ARG "" "COMPONENT;PREFIX" "DEPENDS" ${ARGN})
```

Comment:
We’ve just switched to two space indentation (rather than four).

Reply:
I discussed this issue with Maggie and it seems there is some confusion. The 'pstore' format rule is four space indentation. Are you warning that this function is using two space indentation? If that is the case, you are correct. New patch will have four space indentation.

Reply:
> Are you warning that this function is using two space indentation?

Yes. As you say, the pstore convention is four space indentation.